.buildkite/pipeline.yml

```diff
@@ -0,0 +1,34 @@
+---
+steps:
+  - type: "waiter"
+
+  - name: ":docker:"
+    command: "bin/buildkite bin/ci build"
+    agents:
+      queue: native
+      docker: builder
+
+  - type: "waiter"
+
+  - name: ":pill:"
+    command: "bin/buildkite bin/ci run npm test"
+    agents:
+      queue: elastic
+
+  - type: "waiter"
+
+  - name: ":rocket: Publish"
+    type: "manual"
+    branches: "master"
+
+  - name: ":speech_balloon:"
+    command: "bin/buildkite bin/ci announce releasing-production"
+    branches: "master"
+    agents:
+      queue: elastic
+
+  - name: ":speech_balloon: Publish Treehorn"
+    command: "bin/buildkite bin/ci create-deploy-candidate production"
+    branches: "master"
+    agents:
+      queue: elastic
```

Comment:
lol. I see - totally not there :)  It's a different approach, I tend to try and lead toward `!fixup`ing my changes as related to to the logical commits rather than "fix rubocop" and "correct spelling" type commits (unless the train of thought is material to the pr kinda thing).  I see both in the wild, just my thoughts